src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -10,25 +10,9 @@ interface IOptionSettlementEngine {
     //////////////////////////////////////////////////////////////*/
```

Comment:
Please roll back all changes in this file.

Reply:
Will do. I just didn't want `OptionsWritten` event in the `Exercise events` section, but I will revert and treat this 100% frozen.